command_line_assistant/daemon/http/query.py

```diff
@@ -83,6 +83,14 @@ def submit(payload: dict, config: Config) -> str:
         raise RequestFailedError(
             f"Communication error with the server: {str(exc)}. Please try again in a few minutes."
         ) from exc
+    except OSError as exc:
+        # If the exception string contains the standard RHSM cert path, assume
+        # the system is not registered and raise a very specific error message.
+        if "/etc/pki/consumer/cert.pem" in str(exc):
+            raise RequestFailedError(
+                "System is not registered with Red Hat. Run `subscription-manager register` or `rhc connect` and try again."
```

Comment:
```suggestion
                "System is not registered with a valid Red Hat subscription. Run `subscription-manager register` or `rhc connect` and try again."
```

Reply:
This message was specifically selected with input from UX. I'd want to run this by Jon before changing it again.

Reply:
Reopening to add John suggestion from slack: 

```suggestion
                "The system must be registered to use RHEL Lightspeed. For cloud-based systems, see: https://access.redhat.com/articles/7127962"
```